src/plugins/data/common/field_formats/converters/bytes.ts

```diff
@@ -23,7 +23,9 @@ import { FIELD_FORMAT_IDS } from '../types';
 export class BytesFormat extends NumeralFormat {
   static id = FIELD_FORMAT_IDS.BYTES;
   static title = 'Bytes';
+  static allowsNumericalAggregations = true;
 
   id = BytesFormat.id;
   title = BytesFormat.title;
+  allowsNumericalAggregations = BytesFormat.allowsNumericalAggregations;
```

Comment:
Overall looks ok, However I don't understand why the attribute is set on both statically and as an attribute.
I would opt for either of the options, but not both, unless there's specific reason to do so.

For example, if you prefer the static attribute, you could still access it in `agg_table` like this: `formatter?.constructor.allowsNumericalAggregations`.